### PR TITLE
doc: add tow Bucket Lifecycle RGW Settings

### PR DESCRIPTION
**Pull request by Huber-ming**
Add tow Bucket Lifecycle RGW Settings.

Signed-off-by: Huber-ming <zhangsm01@inspur.com>

## Checklist
- [ ] References tracker ticket
- [ ] Updates documentation if necessary
- [ ] Includes tests for new functionality or reproducer for bug

---

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`

</details>

### CODE REVIEW

**Review by mattbenjamin — COMMENTED** [review]
thanks for adding these;  I think maybe I would put these near or at the top of the Lifecycle Settings section, because they are very basic;  I don't think "Additional Lifecycle Settings" is adding very much, there's nothing really "additional" about these specifically.  That said, there is a tricky aspect to rgw_lc_max_objs--which is that we don't have dynamic resharding for the queues, and it can be unsafe to change the sharding when there are already lifecycle rules present.  There's no current workaround for that, really.

**Comment by Huber-ming** [comment]
jenkins test signed

**Comment by Huber-ming** [comment]
@mattbenjamin  hi, the commit has `Signed-off-by`, why `Signed-off-by` check fails?

**Comment by tchaikov** [comment]
jenkins test signed

**Comment by tchaikov** [comment]
@Huber-ming i think the reason why the check failed was that the title of your commit message reads

> add tow Bucket Lifecycle RGW Settings 

but all changes which only involves files under `doc/` directory should have a commit message whose title starts with "doc".

in this case, the title of the PR fulfills the requirement. but the title of the commit does not. you might need to revise your commit message accordingly.

File: doc/radosgw/config-ref.rst [hunk 1]
```diff
@@ -389,7 +389,22 @@ effectively throughout their lifetime. In past releases Lifecycle processing was
 by single threaded processing. With the Nautilus release this has been addressed and the
 Ceph Object Gateway now allows for parallel thread processing of bucket lifecycles across
 additional Ceph Object Gateway instances and replaces the in-order
-index shard enumeration with a random ordered sequence.
+index shard enumeration with a random ordered sequence. The following settings may added to
+the Ceph configuration file under the ``[client.radosgw.{instance-name}]`` section.
```
Comment (anthonyeleven) [inline doc/radosgw/config-ref.rst:393]: s/file//

File: doc/radosgw/config-ref.rst [hunk 2]
```diff
@@ -389,7 +389,22 @@ effectively throughout their lifetime. In past releases Lifecycle processing was
...
+
+``rgw_enable_lc_threads``
+
+:Description: Enables the lifecycle maintenance thread. This is required on at least one rgw for each zone.
```
Comment (anthonyeleven) [inline doc/radosgw/config-ref.rst:397]: s/rgw/RGW/

**Review by anthonyeleven — CHANGES_REQUESTED** [review]
Requested some minor changes.  Also, the PR and commit titles should be something like

"doc/radosgw: Add two bucket lifecycle settings"

**Pull request closed without merging**